### PR TITLE
Rework Gateway RL Page

### PR DESCRIPTION
**Pull request by mheap**

### CODE REVIEW

File: app/_plugins/filters/slugify.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Jekyll
+  module SlugifyFilter
+    def slugify(input)
+      if input.is_a? String
+        input.downcase.strip.gsub(' ', '-').gsub(/[^\w-]/, '')
+      else
+        raise "Invalid input type: #{input.class} to 'slugify'. Input must be a string."
+      end
```
Comment (fabianrbz) [inline app/_plugins/filters/slugify.rb:10]: @mheap jekyll has a built-in [slugify filter](https://jekyllrb.com/docs/liquid/filters/) :)

File: app/_landing_pages/gateway/rate-limiting.yaml [hunk 2]
```diff
@@ -22,76 +22,95 @@ content:
...
+                  ee: true
+                - title: Fixed window rate limits
+                  oss: true
+                  ee: true
+                - title: Advanced rate limiting algorithms
+                  subtitle: e.g. Leaky bucket, Sliding window
+                  oss: false
+                  ee: true
+                - title: Enhanced window size configuration
+                  subtitle: Choose any interval, not just `second`, `minute` etc
```
Comment (Guaris) [inline app/_landing_pages/gateway/rate-limiting.yaml:53]:
```suggestion
                  subtitle: Configure custom intervals beyond standard options like `second` or `minute`.
```

File: app/_landing_pages/gateway/rate-limiting.yaml [hunk 3]
```diff
@@ -22,76 +22,95 @@ content:
...
+                  oss: false
+                  ee: true
+                - title: Precise counter control
+                  subtitle: Choose if a blocked request contributes towards a rate limit
```
Comment (Guaris) [inline app/_landing_pages/gateway/rate-limiting.yaml:57]:
```suggestion
                  subtitle: Specify whether blocked requests count towards the rate limit calculation.
```

**Review by Guaris — COMMENTED** [review]
I like the way you configured the table, and I also really like the subtitles. 

I don't like the Capabilities header, I think the page doesn't need it and can just start with the first sentence. 

On a note unrelated to this PR and actually related to styling, I wish 'support by:*' was less prominent in these cards:
image

File: app/_landing_pages/gateway/rate-limiting.yaml [hunk 4]
```diff
@@ -22,76 +22,95 @@ content:
...
-
-                    The rate limit counter can be stored in memory, Redis, or in the Postgres database (traditional mode only).
-
-                    Multiple rate limits can be configured, allowing you to specify complex rules such as 1000 requests per hour, but no more than 100 per minute.
+              columns:
+                - title: Rate Limiting
+                  key: oss
+                - title: Rate Limiting Advanced
+                  key: ee
+              features:
```
[the remark this answers is not in the document]
Comment (mheap) [reply app/_landing_pages/gateway/rate-limiting.yaml:35]: What do you mean? In the top left cell?

[the remark this answers is not in the document]
Comment (Guaris) [reply app/_landing_pages/gateway/rate-limiting.yaml:35]: Yeah top left

**Comment by fabianrbz** [comment]
@mheap can we merge this?

**Comment by mheap** [comment]
Good catch! Yes, it should be under the feature table

**Comment by fabianrbz** [comment]
I'll add that

**Pull request merged**